### ISSUE TITLE
文档太少太不详细了，错误都不知道是什么

### ISSUE THREAD

**Issue by teli5200** [alibaba/jetcache#280]
2019-04-14 12:24:00.068 [scheduling-1] ERROR WeTaoMall.service.WeServerService - 信息获取失败:no remote cache builder: default

jetcache:
  statIntervalMinutes: 15
  areaInCacheName: false
  hidePackages: com.alibaba
  local:
    default:
      type: caffeine
      keyConvertor: fastjson
      expireAfterWriteInMillis: 100000
  remote:
    default:
      type: redis.lettuce
      keyConvertor: fastjson
      uri: redis://${spring.redis.password}@${spring.redis.host}:${spring.redis.port}/
      poolConfig:
        minIdle: 5
        maxIdle: 20
        maxTotal: 50
      host: ${spring.redis.host}
      port: ${spring.redis.port}
      password: ${spring.redis.password}

**Comment by teli5200** [comment]
缩进没有问题，这个粘贴过来没有了

**Comment by areyouok** [comment]
出现这个错误是因为你的remote cache没有配置好
